lib/pantry/handlers/intent_request_handler.rb

```diff
@@ -0,0 +1,15 @@
+module Pantry
+
+  module Handlers
+
+    class IntentRequestHandler
+
+      def process( session: )
+
+      end
+
+    end
+
+  end
+
+end
```

Comment:
getting into the nitpicky part of the code review, but we roll with a little less spacing here:
```ruby
module Pantry
  module Handlers
    class IntentRequestHandler

      def process( session: )
      end

    end
  end
end
```